quic/s2n-quic-core/src/recovery/bandwidth/estimator.rs

```diff
@@ -55,6 +59,14 @@ impl core::ops::Mul<Ratio<u64>> for Bandwidth {
     }
 }
 
+impl core::ops::Mul<Duration> for Bandwidth {
+    type Output = u64;
```

Comment:
The output is number of bytes, since Bandwidth is expressing number of bytes over a duration, when you multiple by a duration it goes back to being number of bytes. I may at some point make this a new type (maybe just Bytes(u64)), but with BBRv2 code still in flux I would prefer to tackle that a little later.